src/chrome/content/rules/Equifax.xml

```diff
@@ -0,0 +1,52 @@
+<!--
+	Non-functional hosts
+		Mismatch:
+			- inform.equifax.com
+			- app.inform.equifax.com
+			
+			- consumer.equifax.com
+			- www.consumer.equifax.com
+
+		Incomplete chain:
+			- help.equifax.com
+			- help-en.equifax.ca
+			- help-fr.equifax.ca
+-->
+
+<ruleset name="Equifax">
+
+	<target host="equifax.com" />
+	<target host="www.equifax.com" />
+	<target host="www.ai.equifax.com" />
+	<target host="www.alerts.equifax.com" />
+	<target host="blog.equifax.com" />
+	<target host="aa.econsumer.equifax.com" />
+	<target host="www.econsumer.equifax.com" />
+	<target host="emscert.equifax.com" />
+	<!--<target host="eport.equifax.com" />-->
```

Comment:
Please move this comment to the comments on top and also add a hint about the issue.